website/pages/en/chain-integration-overview.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Chain Integration Process Overview
+---
+
+A transparent and governance-based integration process was designed for blockchain teams seeking integration with The Graph protocol. It is a 3-phase process, as summarised below.
+
+## Graph Chain Integration Process
+
+## Stage 1. Technical Integration
+
+- Teams work on a Graph Node integration. [Here's how](/en/new-chain-integration/).
```

Comment:
Links should not include the locale, because otherwise the translated versions of this page will link to an English page.
```suggestion
- Teams work on a Graph Node integration. [Here's how](/new-chain-integration/).
```